api/README.md

```diff
@@ -1,44 +1,73 @@
-# REST HTTP API INSTRUCTIONS
-
-Description of this functionality can be found in [this](https://github.com/livepeer/verification-classifier/issues/40) github issue.
+# Verification API
+
+The verifier API exposes video verification capabilities through HTTP REST endpoints. 
+
+## API endpoints
+### /verify
+
+#### 
+**Method:** `POST`  
+**Content-type:** `application/json` or `multipart/form-data`  
+**Description:**  
+This function performs verification of renditions of source video. There are two use cases:  
+1. video files are accessible to the server
+            - content-type of the request should be application/json
+2. video files are passed in request body (like a browser)
+            - content-type of the request should be multipart/form-data
+            - parameters should be passed as JSON in form's single 'json' field
+            - files should be passed as multipart data, there should be correspondence between 'source, 'uri' fields in parameters and file names. Name fields of files should be unique.  
+            
+**Parameters:**  
+``` {
+"orchestrator_id": The ID of the orchestrator responsible of transcoding  
+"source": a URI or name of the source video    
+"renditions": a list of renditions with the following structure:  
+    [{
+         "uri": a name or URI of rendition,
+         "resolution":{
+            "height": integer, vertical dimension, in pixels
+            "width": integer, horizontal dimension, in pixels
+         },
+         "frame_rate": float, expected FPS, with a special value of 0, which means 'same as source video' 
+         "pixels": integer, The number of expected total pixels
+                   (height x width x number of frames)
+    }] 
+}
+```
+**Returns:**
+```
+{
+  "orchestrator_id": The ID of the orchestrator responsible of transcoding ,
+   "results":[      {
+         "audio_available": boolean, whether file contained audio stream,
+         "fps": actual rendition framerate,
+         "width": rendition frame width,         
+         "height": rendition frame height,
+         "ocsvm_dist": distance to separating plane of OCSVM classifier for this rendition, negative value represents an outlier (tampered rendition),
+         "tamper": binary classification result, 1 if rendition predicted to be tampered, 0 otherwise
+         "tamper_sl": binary classification result from supervised CatBoost model,
```

Comment:
For both this field and `tamper_ul` can we document that it will be 1 or 0 (to avoid anyone assuming that a boolean is returned) similar to what we do for the `tamper` field?